specification/appendix/commitment_discounts/size_flexibility.md

```diff
@@ -0,0 +1,200 @@
+# Size Flexibility
```

Comment:
Side note: If we call this "size flexibility", I believe this will lock us into `SkuSize` for the thing that conveys the actual size/shape/SKU/edition/etc of the resource.